### DISCUSSION TITLE
Opens a translation for "Tranditional Chinese (Hong Kong)"

### DISCUSSION THREAD

**Discussion by markmybytes** [UnicornsOnLSD/finamp#431] [Translations]
I don't see this option in "Create new translation" on Weblate. It seems it can only be created by admins.
`zh-Hant-HK` for l10n code

**Comment by UnicornsOnLSD** [comment]
You should be able to start a new translation, are you logged in?

**Comment by markmybytes** [comment]
Yes, but 

![圖片](https://user-images.githubusercontent.com/71750702/232060096-3775585a-607f-406a-b860-1d2605f230c5.png)

**Reply by UnicornsOnLSD** [reply]
Ah, that's weird. I've added a zh_Hant_HK translation. Let me know if you have any other issues :)
